src/components/FeaturedProject.tsx

```diff
@@ -31,9 +31,10 @@ function FeaturedProject(props: featuredProjectProps) {
             {props.videoEmbedID ? (
                 <S.FeaturedProjectMediaContainer>
                     <iframe
+                        style={{ pointerEvents: "none" }}
                         width="100%"
                         height="100%"
-                        src={`https://www.youtube.com/embed/${props.videoEmbedID}`}
+                        src={`https://www.youtube.com/embed/${props.videoEmbedID}&autoplay=1&mute=1`}
```

Comment:
Last request - can we enable looping? Right now the video ends and since hover is disabled there is no way to restart it.
![image](https://github.com/techstartucalgary/tsu-website/assets/59885733/c42239ea-16e4-47b6-8667-8f64f23ee432)